### DISCUSSION TITLE
[Enhancement] Is there a way to take snapshot for a xamarin view and It is not display

### DISCUSSION THREAD

**Discussion by llzz19851985** [xamarin/Xamarin.Forms#11698] [General]
## Summary
I have a xamarin view and it is not display,I want to get the view snapshot and convert to android bitmap,is there a way to this target?

xamarin version : 4.2.0.778463

**Comment by PureWeen** [comment]
Hey @llzz19851985 apologies but I'm not quite following your question here.

I think you might have better luck getting this answered on StackOverflow

I'm going to move this issue to discussions for now

If you want to convert a forms view to a bitmap I would look at a few of the links here

https://stackoverflow.com/questions/5536066/convert-view-to-bitmap-on-android
https://docs.microsoft.com/en-us/xamarin/xamarin-forms/app-fundamentals/custom-renderer/

**Comment by llzz19851985** [comment]
@PureWeen A xamarin forms View not diplay can get its render?i know use android native view can convert to bitmap without display,but it is very complicated,than use xamarin forms view convert it is very simple,because i can reuse most of part view!

**Reply by PureWeen** [reply]
You can't snap shop a Xamarin.Forms View that isn't rendered because the native platform is what renderers the views. If you only have a Xamarin.Forms View you can't convert that to a PNG because you would need something to interpret the view to what it would look like visually which is going to be different for each platform.
